monkey/common/events/credentials_stolen_events.py

```diff
@@ -15,4 +19,4 @@ class CredentialsStolenEvent(AbstractEvent):
         :param stolen_credentials: The credentials that were stolen by an agent
     """
 
-    stolen_credentials: Sequence[Credentials]
+    stolen_credentials: Sequence[Credentials] = field(default_factory=hack_event)
```

Comment:
Why are we using this hack? It's perfectly reasonable for `stolen_credentials` to have a default.

```suggestion
    stolen_credentials: Sequence[Credentials] = field(default_factory=list)
```